docs/rbe/rbe.md

```diff
@@ -0,0 +1,161 @@
+# RBE for Flutter Engine Developers
+
+## Overview
+
+This is documentation on setting up RBE for building the Flutter engine. It is
+expected to work for Googlers only on corp Linux, macOS, and Windows hosts,
+including cloudtop instances.
+
+## Getting started
+
+The first step is to add an entry to the `.gclient` file. The entry to add is
+`"use_rbe": True` in the `custom_vars` section. It should look like this:
+
+After making this edit, you must be authenticated as a Googler by CIPD so that
+the RBE configurations can be downloaded from the `flutter_internal`
+[CIPD bucket](https://chrome-infra-packages.appspot.com/p/flutter_internal):
```

Comment:
Should this be https://chrome-infra-packages.appspot.com/p/infra_internal? The linked bucket doesn't exist

Reply:
The link should show up after authenticating with CIPD, I think.